<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAdminWindow</class>
 <widget class="QDialog" name="AddAdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Администратор</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>97</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="NamelineEdit">
       <property name="placeholderText">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NumberlineEdit">
       <property name="placeholderText">
        <string>+7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="MaillineEdit">
       <property name="placeholderText">
        <string>Электронная почта</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LoginlineEdit">
       <property name="placeholderText">
        <string>Логин</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PasswordlineEdit">
       <property name="placeholderText">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="RPasswordlineEdit">
       <property name="placeholderText">
        <string>Повторите пароль</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>97</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="AddAdminpushButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelpushButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
